modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -37,22 +37,19 @@ The Red Hat Virtualization (RHV) environment has the following prerequisites for
 == {VirtProductName} prerequisites
 
 The {VirtProductName} environment has the following prerequisites for VM import:
-
-* You must be an admin user.
-* The {product-title} storage class must be NFS.
 endif::[]
-
 ifdef::virt-importing-vmware-vm[]
-Importing a virtual machine from VMware vCenter into {VirtProductName} has the following prerequisites:
+Importing a VMware VM into {VirtProductName} has the following prerequisites:
 
-* You must be an admin user.
 * You must have access to an image registry.
 * You must create a VMware Virtual Disk Development Kit (VDDK) image, push it to an image registry, and add it to the `v2v-vmware` ConfigMap.
-* The VMware virtual machine must be powered off.
+* The VMware virtual machine must be powered off during import.
```

Comment:
The VMware to CNV VM import do not support import of VMware running VMs.
VM import will fail in such a case with error in the virt-v2v